### DISCUSSION TITLE
How do I set this up and play with it locally on a windows 10 64 bit machine?

### DISCUSSION THREAD

**Discussion by naveenmarthala** [mmistakes/minimal-mistakes#2644] [Q&A] [answered]

**Comment by iBug** [comment, marked as answer]
Read the docs first.

You'll need to follow [the guide](https://jekyllrb.com/docs/installation/windows/) to setup Jekyll first, followed by [this theme's quick-start guide](https://mmistakes.github.io/minimal-mistakes/docs/quick-start-guide/).
